### PR TITLE
v8.15.0

### PR DESCRIPTION
**Pull request by REggebrecht**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -215,6 +221,20 @@
...
+      "dev": true,
+      "requires": {
+        "assertion-error": "^1.1.0",
+        "check-error": "^1.0.2",
+        "deep-eql": "^3.0.1",
+        "get-func-name": "^2.0.0",
+        "pathval": "^1.1.0",
+        "type-detect": "^4.0.5"
+      }
+    },
```
Comment (ceisele-r) [inline package-lock.json]: Please revert the changes here / execute npm i again. I don't think these changes should be in here (except the version number)

Comment (REggebrecht) [reply package-lock.json]: reverted changes
